x-pack/test/case_api_integration/basic/tests/cases/migrations.ts

```diff
@@ -15,41 +15,72 @@ export default function createGetTests({ getService }: FtrProviderContext) {
   const esArchiver = getService('esArchiver');
 
   describe('migrations', () => {
-    before(async () => {
-      await esArchiver.load('cases/migrations');
-    });
+    describe('7.10.0', () => {
+      before(async () => {
+        await esArchiver.load('cases/migrations/7.10.0');
+      });
 
-    after(async () => {
-      await esArchiver.unload('cases/migrations');
-    });
+      after(async () => {
+        await esArchiver.unload('cases/migrations/7.10.0');
+      });
+
+      it('migrates cases connector', async () => {
+        const { body } = await supertest
+          .get(`${CASES_URL}/e1900ac0-017f-11eb-93f8-d161651bf509`)
+          .set('kbn-xsrf', 'true')
+          .send()
+          .expect(200);
+
+        expect(body).key('connector');
+        expect(body).not.key('connector_id');
+        expect(body.connector).to.eql({
+          id: 'connector-1',
+          name: 'none',
+          type: '.none',
+          fields: null,
+        });
+      });
 
-    it('7.10.0 migrates cases connector', async () => {
-      const { body } = await supertest
-        .get(`${CASES_URL}/e1900ac0-017f-11eb-93f8-d161651bf509`)
-        .set('kbn-xsrf', 'true')
-        .send()
-        .expect(200);
-
-      expect(body).key('connector');
-      expect(body).not.key('connector_id');
-      expect(body.connector).to.eql({
-        id: 'connector-1',
-        name: 'none',
-        type: '.none',
-        fields: null,
+      it('migrates cases settings', async () => {
+        const { body } = await supertest
+          .get(`${CASES_URL}/e1900ac0-017f-11eb-93f8-d161651bf509`)
+          .set('kbn-xsrf', 'true')
+          .send()
+          .expect(200);
+
+        expect(body).key('settings');
+        expect(body.settings).to.eql({
+          syncAlerts: true,
+        });
       });
     });
 
-    it('7.11.0 migrates cases settings', async () => {
-      const { body } = await supertest
-        .get(`${CASES_URL}/e1900ac0-017f-11eb-93f8-d161651bf509`)
-        .set('kbn-xsrf', 'true')
-        .send()
-        .expect(200);
+    describe('7.11.1', () => {
```

Comment:
nit: I think `7.11` as it targets the whole transition from `7.11` to `7.12`. `7.11.1` is a bit confusing as it seems that this test in only for `7.11.1` and not `7.11`.

Reply:
Thanks for the review! Yeah I'll add more of a description. I chose `7.11.1` because that's the stack version I used to generate the data so I think I'll leave that as the directory name.